### PR TITLE
Bump version to v2.27

### PR DESCRIPTION
**Pull request by ydah**
If you notice, a lot of changes are stocked in the CHANGELOG, so it's probably time for a release. 
(If possible, v2.28 want [extract the cop of the RSpec/Rails department. ](https://github.com/rubocop/rubocop-rspec/issues/1734))

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [-] Added tests.
- [x] Updated documentation.
- [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
...
 
+## 2.27.0 (2024-03-01)
+
 - Add new `RSpec/IsExpectedSpecify` cop. ([@ydah])
+- Add new `RSpec/RepeatedSubjectCall` cop. ([@drcapulet])
 - Add support for `assert_true` and `assert_false` to `RSpec/Rails/MinitestAssertions`. ([@ydah])
-- Support asserts with messages in `Rspec/BeEmpty`. ([@G-Rath])
 - Add support for `assert_empty`, `assert_not_empty` and `refute_empty` to `RSpec/Rails/MinitestAssertions`. ([@ydah])
 - Support correcting some `*_predicate` assertions in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
 - Support correcting `*_kind_of` assertions in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
```
Comment (bquorning) [inline CHANGELOG.md:12]: I was thinking maybe we should squash all these lines about MinitestAssertion.

Comment (ydah) [reply CHANGELOG.md]: Indeed it is, I updated CHANGELOG.md

**Comment by ydah** [comment]
@bquorning Speaking of which, I don't think I had [permission to publish to rubocop-rspec's Rubygem.org yet](https://rubygems.org/profiles/ydah). Is it possible to add me to owners? Or can you publish on Rubygem.org instead of me?

**Comment by bquorning** [comment]
> Speaking of which, I don't think I had [permission to publish to rubocop-rspec's Rubygem.org yet](https://rubygems.org/profiles/ydah). Is it possible to add me to owners? 

👍🏼 Done

**Review by pirj — APPROVED** [review]
Fantastic!

**Pull request merged**